optaplanner-core/src/test/java/org/optaplanner/core/impl/partitionedsearch/queue/PartitionQueueTest.java

```diff
@@ -31,6 +33,16 @@
 
     private ExecutorService executorService = Executors.newFixedThreadPool(2);
 
+    @After
+    public void tearDown() {
+        executorService.shutdown();
+        try {
+            executorService.awaitTermination(1, TimeUnit.SECONDS);
+        } catch (InterruptedException ex) {
+            throw new RuntimeException(ex);
+        }
+    }
```

Comment:
Looks like it is a good idea to clean up threads after the test. For example PIT says:
```
More threads at end of test (6) addExceptionWithNonEmptyQueue(org.optaplanner.core.impl.partitionedsearch.queue.PartitionQueueTest) than start. (4)
```
IIUC this means that every time the test runs it leaves 2 additional threads (somewhere in memory?). I don't know how much resources these threads can consume but it's nicer to clean them up.